src/state/Maps/actions.js

```diff
@@ -145,6 +145,31 @@ function updateStateFromView(data) {
 }
 
 
+/**
+ * 
+ * It enables to update any layer property except layerKey. 
+ * Layer object is merged with actual layer option.
+ */
+function updateMapLayer(mapKey, layerKey, update) {
+	return (dispatch, getState) => {
+		const state = getState();
+		const mapByKey = Select.maps.getMapByKey(state, mapKey);
+		if(!mapByKey) {
```

Comment:
Isn't check if layer exists enough?

Reply:
mapKey is input parametr. I dont see nothing wrong on test it.